Fakes/OpenActive.FakeDatabase.NET/FakeBookingSystem.cs

```diff
@@ -43,25 +45,23 @@ public InMemorySQLite()
             // in-memory database
             //const string ConnectionString = "Data Source=/Users/nick/repos/openactive/openactive-test-suite/testsuite.db;Version=3;";
             //const string ConnectionString = "Data Source=InMemoryDatabase;Mode=Memory;Cache=Shared;Version=3;";
-            const string ConnectionString = "Data Source=:memory:?cache=shared";
+            const string ConnectionString = ":memory:";
```

Comment:
Seems much simpler! How come this doesn't need `?cache=shared"`?

Reply:
OrmLite automatically registers a `MemoryCacheClient()` if no caching provider is registered explicitly.
ref: [](https://docs.servicestack.net/caching)

Reply:
Interesting... I'd understood previously that without `?cache=shared` each new connection would access a completely new database. Perhaps there's some other magic going on here... no worries!